### PR TITLE
tls: fix cluster TLS while using CR to create cluster

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

this pr fixes: #1771 

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
Fix cluster TLS while using CR to create cluster
```

### CODE REVIEW

File: pkg/apis/pingcap/v1alpha1/tikv_config.go [hunk 1]
```diff
@@ -194,11 +194,11 @@ type TiKVRaftDBConfig struct {
...
 type TiKVSecurityConfig struct {
 	// +optional
-	CAPath string `json:"ca_path,omitempty" toml:"ca_path,omitempty"`
+	CAPath string `json:"ca-path,omitempty" toml:"ca-path,omitempty"`
 	// +optional
-	CertPath string `json:"cert_path,omitempty" toml:"cert_path,omitempty"`
+	CertPath string `json:"cert-path,omitempty" toml:"cert-path,omitempty"`
 	// +optional
-	KeyPath string `json:"key_path,omitempty" toml:"key_path,omitempty"`
+	KeyPath string `json:"key-path,omitempty" toml:"key-path,omitempty"`
```
Comment (weekface) [inline pkg/apis/pingcap/v1alpha1/tikv_config.go]: There are many other keys are connected with `_`, do they should be change to `-`? @aylei PTAL

**Comment by cofyc** [comment]
`Scaling tidb cluster with advanced statefulset ` this is a flaky test. It happened frequently. I'll look at it.

**Pull request merged**